compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -130,6 +136,8 @@ int entry(int argc, char **argv)
     options->enable(Algorithms::ResolveCustomOpBatchMatMul);
     options->enable(Algorithms::ResolveCustomOpMatMul);
   }
+  if (arser.get<bool>("--fuse_add_with_tconv"))
+    options->enable(Algorithms::FuseAddWithTConv);
```

Comment:
Is it possible to add also to `--all` ?

Reply:
@mhs4670go , was there any particular reason for `fuse_batchnorm_with_tconv` not added to `all` ?

Reply:
@seanshpark When we were discussing the pass, we decided to add it to `--all` section as the path became more mature. There are still few places to use this path, and I'm gonna put it in after some more tests are added, which by the way, is the reason why I don't close related issue yet.

Reply:
OK, thanks!